### PR TITLE
net-misc/freerdp: Include news deps and misc fixes

### PR DESCRIPTION
**Pull request by inode64**
* Include support for ICU and GSM
* change http to https
* Disable OOS to prevents RDP connection from crashing when OSS is disabled in the kernel and starts playing a sound
* Disable developer warnings

Package-Manager: Portage-3.0.41-r, pkgcheck-0.10.20
Signed-off-by: Fco Javier Felix <ffelix@inode64.com>

### CODE REVIEW

**Review by floppym — CHANGES_REQUESTED** [review]
Please do not mess with the ordering of existing cmake args. They are currently sorted by USE flag name, and I would like to keep it that way.

File: net-misc/freerdp/freerdp-2.9.0.ebuild [hunk 1]
```diff
@@ -67,6 +68,7 @@ RDEPEND="
 			x11-libs/libXtst
 			xinerama? ( x11-libs/libXinerama )
 		)
+		ffmpeg? ( media-video/ffmpeg[encode,x264] )
```
Comment (floppym) [inline net-misc/freerdp/freerdp-2.9.0.ebuild:71]: Why are you adding this dependency?

Comment (inode64) [reply net-misc/freerdp/freerdp-2.9.0.ebuild:71]: ffmpeg is used to encode audio, if this is not enabled, the freerdp server cannot stream audio.
Look at the code in libfreerdp/codec/dsp_ffmpeg.c , class freerdp_dsp_ffmpeg_encode for example

Comment (jospezial) [reply net-misc/freerdp/freerdp-2.9.0.ebuild:71]: See also
https://github.com/FreeRDP/FreeRDP/issues/8686
build error with ffmpeg master git: libfreerdp/codec/h264_ffmpeg.c:618:55: error: ‘AV_CODEC_CAP_TRUNCATED’ undeclared

**Review by floppym — CHANGES_REQUESTED** [review]
Please rework this branch so that each commit makes a single logical change.

Also, please update the commit messages to comply with [GLEP 66](https://www.gentoo.org/glep/glep-0066.html#commit-messages).

**Comment by jospezial** [comment]
See also https://bugs.gentoo.org/909508
dev-libs/pkcs11-helper and sdl deps

**Comment by jospezial** [comment]
https://bugs.gentoo.org/913391
net-libs/webkit-gtk webkit2gtk-4.0 or QtWebEngine dep

Unbundle https://github.com/webview/webview possible?

**Comment by floppym** [comment]
I have incorporated most of the changes from this PR.

**Pull request closed without merging**